lib/config.js

```diff
@@ -1,12 +1,12 @@
 try {
   require('dotenv').config()
-  var fs = require('fs');
-  var homeDir = require('os').homedir();
+  const fs = require('fs');
+  const homeDir = require('os').homedir();
 } catch(err) {
```

Comment:
Let's remove this try / catch block

Reply:
Cool, I'll roll this change in with the test coverage update. I have some work done on that already that I can just PR to get the project moving, then we can circle back to adding more test coverage when we have the bandwidth available.